dist/rpm/openQA.spec

```diff
@@ -387,6 +387,11 @@ export LANG=en_US.UTF-8
 rm -rf %{buildroot}/%{_sysusersdir}
 %endif
 
+%if 0%{?suse_version} > 1560
```

Comment:
what happens in other distros? this will be always false, right?

Reply:
I think it'll be true on the corresponding SLE but on other distros like Fedora it will be always false. That's not a problem, we just don't package that file then which is the conservative approach.